source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,129 @@ Authentication Connection Tab
    :class: singlecol
 
 
+The :guilabel:`Authentication` tab allows you to connect deployments that 
+require authentication. To learn about authentication mechanisms within MongoDB,
+see :driver:`Authentication Mechanisms </go/current/fundamentals/auth/>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Authentication` tab.
+
+      Select your authentication method from the following options: 
+      
+      - :ref:`Username/Password <username-password>`
+      
+      - :ref:`X.509 <x509>`
+      
+      - :ref:`Kerberos <kerberos-authentication>`
+      
+      - :ref:`LDAP <ldap-authentication>`
+      
+      - :ref:`AWS IAM <aws-iam-authentication>`
+
+      .. _username-password:
+
+      Username/Password 
+      ~~~~~~~~~~~~~~~~~
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      - (Optional) :manual:`Authentication Database </core/security-users/#authentication-database>`
+
+      - Select an Authentication Mechancism:
+         
+        - Default
+
+          MongoDB uses :manual:`SCRAM </core/security-scram/>` as the default 
+          authentication mechanism.
+
+        - :manual:`SCRAM-SHA-1 </core/security-scram/>`
+
+        - :manual:`SCRAM-SHA-256 </core/security-scram/>`
```

Comment:
I'm a little confused what the "Default" is. We say the default is "SCRAM", but when I get to the SCRAM docs page, I see SCRAM-SHA-1 and SCRAM-SHA-256 explained. It's unclear (to me) which SCRAM is the default.

Reply:
Done.